itests/app_orjson.py

```diff
@@ -0,0 +1,65 @@
+import uuid
+import base64
+import uvicorn
+import orjson
+import dataclasses
+from datetime import datetime
+
+from blacksheep.server import Application
+from blacksheep.server.responses import json
+from blacksheep.plugins import Plugins, JSONPlugin
+
+
+def orjson_dumps(obj):
+    def default(x):
+        if isinstance(x, bytes):
+            return base64.urlsafe_b64encode(x).decode('utf-8')
+
+        raise TypeError
+
+    return orjson.dumps(obj, default=default).decode('utf-8')
+
+
+app_orjson = Application(
```

Comment:
Hi, a made some fixes, and re-write plugins to use them as non-global objects, they now can be configured in such way

Reply:
Hi! Thanks again for your contribution - after reviewing your latest changes, I decided to try an alternative way. I didn't want to annoy you asking to try another thing, so I forked your fork and tried myself

1. I confirm that with the code you wrote initially, the whole test suite was running with specific JSON settings using `orjson`, not only the application whose variable is named `app_orjson`
2. to keep the JSON settings global, which is fine for regular users of the framework and only not fine in the context of the test suite for BlackSheep, I leveraged instead the principle of process forking, so I slightly modified your code to apply specific JSON settings only in the process that runs `app_orjson`

In other words, instead of making many changes to the existing code to support non-global JSON settings, I tried to apply specific JSON settings only to the new tests you added. I like this solution more.

I will create a second PR, where we will result both as contributors. 😄